modular_skyrat/modules/verbs/code/subtle.dm

```diff
@@ -13,6 +13,9 @@
 	return FALSE
 
 /datum/emote/living/subtle/run_emote(mob/user, params, type_override = null)
+	if(!can_run_emote(user))
+		user.show_message(span_warning("You can't emote at this time."))
```

Comment:
Why `show_message` over `to_chat`?

Reply:
Cause there's 10 different ways to send a message to a player, and I can never remember the right one.